src/Routers/CloudCodeRouter.js

```diff
@@ -102,7 +103,9 @@ export class CloudCodeRouter extends PromiseRouter {
         req.auth,
         '_JobSchedule',
         { objectId },
-        formatJobSchedule(job_schedule)
+        formatJobSchedule(job_schedule),
+        null,
```

Comment:
Can you set the clientSDK to `undefined` instead of `null` so we preserve the original value.

Reply:
done